project/views.py

```diff
@@ -73,14 +74,19 @@ def project_details(request, project_id):
 @videssur_required
 def project_list(request):
     objects = Project.objects.filter(ong=request.user.ong).values()
+
+
     form = FilterProjectForm(request.GET or None)
 
     if request.method == 'GET':
         objects = project_filter(objects, form)
 
+    paginator = Paginator(objects, 12)
```

Comment:
The json that is passed to the view should be made up of the projects on the page itself. To implement this you should have something like:

```
paginator = Paginator(objects, 12)
page_number = request.GET.get('page')
project_page = paginator.get_page(page_number)

projects_dict = [project for project in project_page]
for p in projects_dict:
    p.pop('_state', None)

project_json = json.dumps(projects_dict, cls=CustomJSONEncoder)

context = {
    'objects': project_page,
    'objects_json' : project_json,
    'object_name': 'proyecto',
    'object_name_en': 'project',
    'title': 'Gestión de Proyectos',
    'form': form,
}
```